fluentd/run.sh

```diff
@@ -16,10 +16,12 @@ for file in ${ocp_fluentd_files} ; do
     sed -i -e "s/%OCP_FLUENTD_TAGS%/${OCP_FLUENTD_TAGS}/" $file
 done
 
+mkdir -p `dirname $LOGGING_FILE_PATH`; touch $LOGGING_FILE_PATH; exec >> $LOGGING_FILE_PATH 2>&1
```

Comment:
This has to go after line 25, after the last `echo "============================="`
We want the notice about the the `$LOGGING_FILE_PATH` and the `logs` command to be available via `oc logs $fluentd_pod`.  Everything after that, we do not want to be available via `oc logs $fluentd_pod`, only via `logs` or `$LOGGING_FILE_PATH`

Reply:
Ahh, I see.  Sorry.   Fixed.